src/platform-includes/getting-started-config/javascript.deno.mdx

```diff
@@ -0,0 +1,19 @@
+```javascript
+// Import from the Deno registry
+import * as Sentry from "https://deno.land/x/sentry/index.mjs";
+
+// or import from npm registry
+import * as Sentry from "npm:@sentry/deno";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  // ...
+});
+```
+
+To ensure the SDK can send events, you should enable network access for your
+ingestion domain:
+
+```bash
+deno run index.ts --allow-net=___ORG_INGEST_DOMAIN___
```

Comment:
Can we provide an example of what `___ORG_INGEST_DOMAIN___` should be?

Reply:
This is one of those strings that gets automatically replaced with the customer ingest domain!

<img width="774" alt="image" src="https://github.com/getsentry/sentry-docs/assets/1150298/6097485d-1965-47c8-93b2-68290d473e67">

Reply:
oh man, my bad. Looks good then!